app/views/layouts/application.html.slim

```diff
@@ -1,3 +1,5 @@
+- content_for :html_lang
+  = params[:locale] || I18n.default_locale
```

Comment:
Why do this ?  you are already normalizing params[:locale] in the application controller and assigning the value to I18n.locale ?  So in this case, you could just use I18n.locale to get this value ?

Reply:
Good point - fixed in 8d93eee